acceptance/openstack/ims/create_image_obs_test.go

```diff
@@ -0,0 +1,62 @@
+package ims
```

Comment:
did you have tests for v1? I see here only v2

Reply:
Yes. It's already there.